### ISSUE TITLE
Where's the dist directory?

### ISSUE THREAD

**Issue by robmaurizi** [cognitom/symbols-for-sketch#31]
Maybe a dumb question, but not being a Sketch, gulp or NPM user for the most part, where's the dist directory that the CSS and fonts are supposed to be dumped in?

**Comment by peacepostman** [comment]
Hello, there is no dumb questions 😊. The dist folder will be automatically created when the task is done and you will found all your icons in it.

Have fun !

**Comment by robmaurizi** [comment]
it doesn't seem to be getting created when the task is done... It'll live in the same directory as the Sketch file (14px)?

**Comment by peacepostman** [comment]
No, unless you change it in the gulpfile, the folder `dist` is supposed to appear on the root of your working directory.

**Comment by ghost** [comment]
will this work if i run npm on windows 
was just trying that but its not working as you are just processing sketch file and env is node so it should work thats what i think correct me if i am wrong

**Comment by peacepostman** [comment]
Why would you run this on windows ? Sketch is still not available on windows so i guess its normal that it doesn't work :)

**Comment by suprob** [comment]
I'm actually having the same issue and I'm on mac. I followed the instructions and no folders are being generated in my working folder. When I do npm run symbols I get two lines in the terminal, one says "Using gulpfile /Users/blah/blah/gulpfile.js" then the other syays "Starting 'symbols'...". Nothing happens after that. Do I need to rename the sketch file or do something with it?

**Comment by jordenchang55** [comment]
Same here.
I followed the instruction, and 'dist' directory was not generated after `npm run symbol`.
It just show "Starting 'symbols'..." then finished without any message.
I run this on Mac with node.js v8.11.1.

**Comment by shaozict** [comment]
export PATH=$PATH:/Applications/Sketch-52.1.app/Contents/Resources/sketchtool/bin
